### ISSUE TITLE
Advanced voidsuit Insulated gloves does not protect against wire splicing shock.

### ISSUE THREAD

**Issue by Acryonic** [Eclipse-Station/NEV-Northern-Light#242]
as title states, the Advanced Voidsuit Insulated gloves that the chief engineers hardsuit has does not protect against electric shock coming from removing a wire splicing.

Found the bug when I attempted to remove a wire splicing in maintainance with the hardsuit enabled and deployed.

The hardsuit gloves were covering a pair of normal insulated gloves, and when I used an armature cutter to cut the wire splicing, it burnt out the left hand prosthethic I was holding the tool with.

I undeployed the suit afterwards and used the armature cutter with only the regular insulated gloves, and it worked as intended then, no shocks, splicing was removed.

**Comment by ShadzFen** [comment]
I tried to fix this issue, but for the life of me I couldn't figure out why it didn't work. All the values are fine as far as I'm aware. 

I tried both with and without insulated gloves underneath the voidsuit. I tried with the voidsuit both deployed and not deployed. I even tried messing with the values ingame, and it didn't seem to want to work.
![image](https://user-images.githubusercontent.com/45645502/86274109-785ad880-bbd1-11ea-8df6-c40d8912cb38.png)


Some potentially useful links for anyone else who tries to fix this:
This is the voidsuit.
https://github.com/Eclipse-Station/NEV-Northern-Light/blob/d98aab1b6d05cb43da7b85d9160042f462c927fb/code/modules/clothing/spacesuits/rig/suits/station.dm#L106

Maybe this could help? I dunno... A lot of hands and feet stuff got removed from Eris and could be missing on something similar to this. I tried setting `|HANDS|FEET` and `|HAND_LEFT|HAND_RIGHT` to no avail.
https://github.com/Eclipse-Station/NEV-Northern-Light/blob/d98aab1b6d05cb43da7b85d9160042f462c927fb/code/modules/clothing/suits/armor.dm#L329

`siemens_coefficient = 0` means electricity doesn't conduct. That's what is already set to the gloves, yet it still conducts it. I imagine there's some sort of code somewhere hidden under a dumb name that is overwriting it somehow. GLHF
